types/jsreport__nodejs-client/jsreport__nodejs-client-tests.ts

```diff
@@ -0,0 +1,18 @@
+import JsReport = require('jsreport');
+import Client = require('@jsreport/nodejs-client');
```

Comment:
```suggestion
import client = require('@jsreport/nodejs-client');
```
https://github.com/jsreport/jsreport/blob/master/packages/nodejs-client/test/clientTest.js#L8